apps/koala-frontend/src/app/features/sessions/components/annotation/annotation.component.ts

```diff
@@ -42,6 +42,7 @@ export class AnnotationComponent implements AfterViewInit, OnChanges, OnDestroy
   @Input() d3ContainerID = 0;
   @Input() displayMode = false;
   @Input() enableAnnotationDelete = false;
+  @Input() deacitvateAnnotationDelete = false;
```

Comment:
Why not using enableAnnotationDelete? It's currently used for the session setting, but serves the same purpose
If there is a reason to separate it, there is a small typo in deactivate

Reply:
because enableAnnotationDelete removes the whole html elements and thus expands/contracts the whole wavesurfer container on play/pause. So this cannot be used